docs/api.md

```diff
@@ -13,11 +13,11 @@ Instead, it *returns* a new, connected component class, for you to use.
 
      - `url` *(String)*: Required. HTTP URL from which to fetch the data.
      - `method` *(String)*: HTTP method. Defaults to `GET`.
-     - `headers` *(Object)*: HTTP headers as simple key-value pairs. Defaults to `Accept` and `Content-Type` set to `application/json`.
-     - `credentials` *(String)*: Policy for credential to include with request. One of `omit`, `same-origin`, `include`. Defaults to `same-origin`. See [`Request.credentials`](https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials) for details.
-     - `redirect` *(String)*: The redirect mode to use: `follow`, `error`, or `manual`. Defaults to `follow`.
+     - `headers` *(Object)*: HTTP headers as simple key-value pairs. Does _not_ support the [`Headers`](https://developer.mozilla.org/en-US/docs/Web/API/Headers) object format. Defaults to `Accept` and `Content-Type` set to `application/json`.
```

Comment:
thanks for doc'ing this. perhaps it should support `Headers` in the future, but I kept it as simple key-values because everything else was at that point.